service/storage_service.cc

```diff
@@ -2079,6 +2084,19 @@ future<> storage_service::raft_state_monitor_fiber(raft::server& raft, cdc::gene
 
 
 future<> storage_service::raft_replace(raft::server& raft_server, raft::server_id replaced_id, gms::inet_address replaced_ip) {
+    auto ignore_nodes_strs = utils::split_comma_separated_list(_db.local().get_config().ignore_dead_nodes_for_replace());
+    std::unordered_set<raft::server_id> ignored_ids;
+    for (const sstring& n : ignore_nodes_strs) {
+        raft::server_id ignored_id;
+        // FIXME: IPs should be supported, but the current implementation of raft_address_map doesn't allow this.
```

Comment:
We can also deprecate the use of IPs with consistent topology.

Reply:
I added this to the FIXME description.